configs/firewalld/helk.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>helk</short>
+  <description>Firewall service for the HELK.</description>
+  <port protocol="tcp" port="9092" />
```

Comment:
we should add ports for nginx/juypter and logstash and other kafka port if it ever gets used:
ports as follows:
nginx:
443
8088
logstash:
3515
5044
8531
kafka:
9092
9093

Reply:
I will add all these ports but I think I will work on a newer version of the script with different versions of the service. Like this we will prevent ports from being open for nothing ;)